lib/ManifestationMaster.inc

```diff
@@ -23,6 +23,10 @@ class ManifestationMaster {
     return isset($this->fields->publicationDetails) ? $this->fields->publicationDetails : NULL;
   }
 
+  private function _getTracks() {
+    return isset($this->fields->tracks) ? $this->fields->tracks : NULL;
```

Comment:
Hvis du vil nørde kunne du omskrive denne til: 
`return $this->fields->tracks ?? NULL`